music21/musicxml/xmlToM21.py

```diff
@@ -3387,6 +3370,16 @@ def xmlTechnicalToArticulation(self, mxObj):
                 #     mallet lift, mallet table, martellato, martellato lift,
                 #     muted martellato, pluck lift, and swing.
                 tech.displayText = mxObj.text
+            if tag == 'fret':
+                try:
+                    tech.number = int(mxObj.text)
+                except (ValueError, TypeError) as unused_err:
+                    pass
+            if tag == 'string':
```

Comment:
line 3373 could be written as `if tag in ('fret', 'string'):` to avoid the duplication of lines.